src/views/Home.vue

```diff
@@ -253,6 +278,17 @@ export default {
         this.pickedHeadline %= Math.min(this.headline.length, 5)
       }, this.HEADLINEINTERVAL)
       this.pickedHeadline = 0
+    },
+    getLoginStatus() {
+      return this.$store.getters.isLogin
+    },
```

Comment:
`this.$store.getters.isLogin` can be assigned as a computed property. See https://vuejs.org/guide/essentials/computed.html#basic-example and https://v3.vuex.vuejs.org/guide/getters.html#the-mapgetters-helper

Reply:
Done.